INSTALL.md

```diff
@@ -1,115 +1,226 @@
+
+## Install miniconda
+
+Download the appropriate installer [here](http://conda.pydata.org/miniconda.html) corresponding to your operating system and execute it.
+
+Ensure that your `~/.bash_profile` (for default bash users) and/or `~/.zshrc` (for zshell users) contains the following line, which prepends a path leading to miniconda's binaries to the `$PATH` variable.
+
+```bash
+# added by Miniconda2 installer
+export PATH="/Users/hannes/miniconda2/bin:$PATH"
+```
+
+This essentially allows miniconda's binaries representing each of your virtual environments to take precedence over any custom, pre-existing installations of pip or python you might have installed through homebrew, for example – thus you do *not* have to uninstall them.
+
+Next, ensure conda is up-to-date.
+
+```bash
+conda update conda
+```
+
+Finally, restart your shell or reload your profile/rc file to ensure your `$PATH` variable was updated:
+
+````bash
+source ~/.bash_profile
+````
+
+## Setup miniconda environment for QIITA
+
+To ensure that QIITA can install its dependencies – some of which require older or specific distributions of pip packages – without interfering with any pre-existing, globally installed pip packages you might have, you will setup a virtual environment in conda that you must *activate* whenever you are working with or running QIITA.
+
+```bash
+conda create --yes --name qiita python=2.7 pip nose flake8 pyzmq networkx pyparsing natsort mock future libgfortran seaborn 'pandas>=0.18' 'matplotlib>=1.1.0' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1' --channel https://conda.anaconda.org/OpenMDAO
+```
+
+If you receive an error message about conda being unable to find one of the specified packages in its repository, you will have to manually search for them (see troubleshooting miniconda below).
+
+Next, activate your newly created virtual environment for qiita:
+
+```bash
+source activate qiita
+```
+
+(When you want to deactivate this environment, e.g. to return to a different project or back to your global python and pip packages, run `source deactivate`)
+
+If your new conda environment is functioning correctly, you should see this kind of output when you run `which python`, indicating that the `python` command now refers to the python binary in your new virtual environment, rather than a previous global default such as `/usr/bin/python`:
+
+```
+▶ which python
+/Users/your_username/miniconda2/envs/qiita/bin/python
+(qiita)
+```
+
+If you don't see this output, your `$PATH` variable was setup incorrectly or you haven't restarted your shell.
+
+As long as you are in the active qiita environment, commands such as `pip install` or `python` will refer to and be contained within this virtual environment.
+
 
 Install the non-python dependencies
```

Comment:
Not sure what you mean – the highlighted section is only discussing how to activate and deactivate qiita conda env, no?